### PR TITLE
Projects feed new

### PR DESCRIPTION
**Pull request by itexpert120**
resolves https://github.com/NEARBuilders/gateway/issues/311

https://github.com/NEARBuilders/gateway/assets/46527178/d20294be-aa56-453e-bb98-0d18a631f546

Current Folder Structure:
image

### CODE REVIEW

File: apps/new/widget/app/view.jsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  config.layout?.src ?? "${alias_devs}/widget/Layout",
+) || { Layout: () => <></> };
+
+// While something like Theme should be in the parent...
+const CSS = styled.div`
+  .container {
+    border: 1px solid red;
+  }
+
+  .button {
```
Comment (saadiqbal-dev) [inline apps/new/widget/app/view.jsx:54]: unused blocks

File: apps/new/widget/app/view.jsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+  .layout {
+    border: 4px solid var(--main-color);
+  }
+
+  .header {
+    border: 1px solid blue;
+  }
+
+  .content {
```
Comment (saadiqbal-dev) [inline apps/new/widget/app/view.jsx:68]: unused blocks

File: apps/new/widget/dummy.jsx [hunk 3]
```diff
@@ -0,0 +1 @@
+return <div>dummy</div>;
```
Comment (saadiqbal-dev) [inline apps/new/widget/dummy.jsx:1]: delete dummy file

**Comment by elliotBraem** [comment]
Other changes we should make:

- [ ] grid vs list view toggle should use state, then debounce update to Storage API
- [ ] projects/list and projects/card are both cards -- let's combine into one with two different variants, or choose better naming (list is misleading)
- [ ] define heading and subheading CSS in the root -- something like "font-family" should be global

**Review by Megha-Dev-19 — COMMENTED** [review]
- I have trouble understanding the naming convention of files
in home you have used `Build`, in projects and components you have used `card` 
I think we decided to use with `PascalCase` right for all the components? like a react component generally have, and the folders in `lowercase with hyphens`
- Can you also please upload the video of change, so we know how it looks?

File: apps/new/widget/dummy.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (itexpert120) [reply apps/new/widget/dummy.jsx]: For pages that are not yet created, just ignore for now.

File: apps/new/widget/page/projects/index.jsx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+      />
+    ),
+    Footer: () => <></>,
+  },
+  router: {
+    param: "tab",
+    routes: {
+      projects: {
+        label: "BUILD",
+        init: {
```
[the remark this answers is not in the document]
Comment (itexpert120) [reply apps/new/widget/page/projects/index.jsx]: We can create an issue for it.

**Comment by itexpert120** [comment]
> * I have trouble understanding the naming convention of files
>   in home you have used `Build`, in projects and components you have used `card`
>   I think we decided to use with `PascalCase` right for all the components? like a react component generally have, and the folders in `lowercase with hyphens`
> * Can you also please upload the video of change, so we know how it looks?

I have fixed the file and folder naming and a view has been added.

File: apps/new/widget/page/home/Build.jsx [hunk 5]
```diff
@@ -294,13 +294,13 @@ const Build = () => {
           </Text>
           <BosVector src={bosImage} />
         </BosContainer>
-        {/* <DesktopContainer> */}
         <GettingStarted>
           <p>Getting Started</p>
           <Button
             style={{
               gap: "8px",
               borderRadius: "10px",
```
Comment (Megha-Dev-19) [inline apps/new/widget/page/home/Build.jsx:302]: does classes work?
`className = "rounded-2 gap-3"`

**Review by Megha-Dev-19 — APPROVED** [review]

File: apps/new/widget/page/projects/index.jsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (Megha-Dev-19) [reply apps/new/widget/page/projects/index.jsx]: can you please create it.?

File: apps/new/widget/dummy.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (Megha-Dev-19) [reply apps/new/widget/dummy.jsx]: can we add a comment in this file about this? so its' understood by others

File: apps/new/widget/page/home/Build.jsx (hunk 5 above)
Comment (itexpert120) [reply apps/new/widget/page/home/Build.jsx:302]: Done

File: apps/new/widget/dummy.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (itexpert120) [reply apps/new/widget/dummy.jsx]: sure

File: apps/new/widget/page/projects/index.jsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (itexpert120) [reply apps/new/widget/page/projects/index.jsx]: Don't we already have an issue for utils?

**Comment by itexpert120** [comment]
> @itexpert120 can we how 2 items/cards per row? would that look better? I think it looks a bit congested right now, or should we leave it right now until the sidebar navbar is finalised?
> 
> Also why do we have `project` inside components? we have `Cards` component inside `home` in `pages`, since it's specific to home page I believe the list and cards are specific to projects itself, isn't it?

We can further discuss it later in tomorrow's meeting. This PR is just for implementing the designs.

**Pull request merged**